mlptrain/sampling/reaction_coord.py

```diff
@@ -122,3 +128,82 @@ def _grad(self, atoms: ase.atoms.Atoms):
     def n_pairs(self) -> int:
         """Number of pairs of atoms in this set"""
         return len(self.atom_pair_list)
+
+
+class DifferenceDistance(ReactionCoordinate):
```

Comment:
Not sure I like this name that much?

Reply:
neither do I, but not sure I have a better option 😂. Might be nice to have a lambda function so you can define any function of distances but also _effort_